### ISSUE TITLE
Error during template rendering

### ISSUE THREAD

**Issue by coreycasmedes** [awesto/django-shop#736]
![screenshot from 2018-06-09 12-00-09](https://user-images.githubusercontent.com/29850702/41194145-bf6ec106-6bdc-11e8-8be7-0e8b4dd6f6b7.png)
I am trying to narrow down this issue and have tried changing to path name but that did not fix the problem.
It seems like it could be a problem with my bootstrap-sass install, so i tried installing them using "pip install django-bootstrap-sass" and was unsuccessful:
Failed to build django-bootstrap-sass
django-haystack 2.5.0 has requirement Django<1.10, but you'll have django 1.11.13 which is incompatible.
drf-haystack 1.6.1 has requirement Django<1.11,>=1.8, but you'll have django 1.11.13 which is incompatible.
These seem like some conflicting requirements.
Im currently running django version 1.11.13.
I am going to keep troubleshooting and will update if I find solution.

**Comment by coreycasmedes** [comment]
Edit: I have tried using relative and absolute path names with no luck. I believe it is something with the bootstrap-sass install since it seems to not be able to find the targetted files.

**Comment by jrief** [comment]
Please use the latest Django-1.11
You also must install bootstrap-sass, available via `npm i bootstrap-sass@3.3.7`.
Make sure, that your `settings.py` contains

```
NODE_MODULES_URL = STATIC_URL + 'node_modules/'

SASS_PROCESSOR_INCLUDE_DIRS = [
    '/path/to/your/project/node_modules',
]
```

**Comment by coreycasmedes** [comment]
I tried what you recommended and still have the same issue.

It seems to be a problem with the path ../bootstrap_sass/assets/stylesheets/bootsrap/variables when I navigate to the indicated path only the _variables.scss exists. I am unsure if this is contributing to the path error problem or not.

Also everytime I reopen the terminal and virtualenv I must run the following line again: $ export DJANGO_SHOP_TUTORIAL=commodity DJANGO_DEBUG=1

or else i get an UncompressableFileError.
 
I also found that if you encounter any problem with a missing cms cascade module you should install them using the following command: $ pip install djangocms-cascade

**Comment by jrief** [comment]
If you look at the file `shop/static/shop/css/_variables.scss` there the import statement is written as

```
@import "bootstrap-sass/assets/stylesheets/bootstrap/variables";
```
from reading your stack trace you prefixed it with a `~`. Maybe this is your problem.

**Comment by jrief** [comment]
If Django-compressor throws a `UncompressableFileError` then this typically is caused, because you added an
```
{% addtoblock "css" %}<link href="https://example.org/some/css/file.css" rel="stylesheet" type="text/css" />{% endaddtoblock %}
```

This however can be fixed by using a special post-compressor. See `shop/sekizai_processors/compress` or https://github.com/django-compressor/django-compressor/pull/891 for details.

**Comment by coreycasmedes** [comment]
I am encountering several import errors with _django-shop.scss and _variables.scss.
I switched the pathname back as you recommended. After that I encountered it couldn't find the awesomefonts files to import with a similar path error. I was able to fix this by downloading awesomefont from their website and add a new import statement. 

Now in the same _django-shop.scss file I am having compile error on line 9: @import 'django-angular';
I have verified that I have all the files downloaded. Any ideas what could be causing all the import issues? 

Also I was encountering the UncompressableFileError just from trying the tutorial. I had not changed any of the code at that point. Running the export statement again in the terminal once you are inside the virtualenv seems to do the trick.
